A770 test_tensor_atomic_rmw failing on nightly run

```
FAILED language/test_core.py::test_tensor_atomic_rmw[shape3-0-1-float64] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_tensor_atomic_rmw[shape7-1-1-float64] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_tensor_atomic_rmw[shape11-0-1-float64] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_tensor_atomic_rmw[shape15-1-1-float64] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_tensor_atomic_rmw[shape19-0-1-float64] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_tensor_atomic_rmw[shape23-1-1-float64] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_tensor_atomic_rmw[shape27-0-1-float64] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_tensor_atomic_rmw[shape31-1-1-float64] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_tensor_atomic_rmw[shape35-0-1-float64] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_tensor_atomic_rmw[shape39-1-1-float64] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_tensor_atomic_rmw[shape43-0-1-float64] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_tensor_atomic_rmw[shape47-1-1-float64] - RuntimeError: Triton Error [ZE]: 0x70000004
```

--- a/python/test/unit/language/test_core.py
+++ b/python/test/unit/language/test_core.py
@@ -1489,6 +1489,9 @@ def test_atomic_rmw_predicate(num_ctas, device):
                           for num_ctas in num_ctas_list
                           for dtype_x_str in ['float32', 'uint64', 'int64', 'float64']])
 def test_tensor_atomic_rmw(shape, axis, num_ctas, dtype_x_str, device):
+    if device == "xpu" and dtype_x_str in (
+            "float64", ) and not triton.runtime.driver.active.get_current_target().arch['has_fp64']:
+        pytest.xfail("float64 not supported on current xpu hardware")
     shape0, shape1 = shape
     # triton kernel
 

--- a/python/test/unit/language/test_core.py
+++ b/python/test/unit/language/test_core.py
@@ -1489,9 +1489,7 @@ def test_atomic_rmw_predicate(num_ctas, device):
                           for num_ctas in num_ctas_list
                           for dtype_x_str in ['float32', 'uint64', 'int64', 'float64']])
 def test_tensor_atomic_rmw(shape, axis, num_ctas, dtype_x_str, device):
-    if device == "xpu" and dtype_x_str in (
-            "float64", ) and not triton.runtime.driver.active.get_current_target().arch['has_fp64']:
-        pytest.xfail("float64 not supported on current xpu hardware")
+    check_type_supported(dtype_x_str, device)
     shape0, shape1 = shape
     # triton kernel
 

